<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGestionLot</class>
 <widget class="QDialog" name="DialogGestionLot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion lot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabRayons">
      <attribute name="title">
       <string>Rayons</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidgetRayons">
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Libellé</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTypesProduits">
      <attribute name="title">
       <string>Types Produits</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidgetTypesProduits">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Types de produit</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rayon</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabProduits">
      <attribute name="title">
       <string>Produits</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidgetProduits">
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <column>
          <property name="text">
           <string>Produit</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type de produit</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLots">
      <attribute name="title">
       <string>Lots</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidgetLot">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabLots">
          <attribute name="title">
           <string>Les lots</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButtonSupprLot">
               <property name="text">
                <string>Supprimer un lot</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QTableWidget" name="tableWidgetLot">
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>160</number>
             </attribute>
             <column>
              <property name="text">
               <string>Produit</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Producteur</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Quantité</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Unité</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Prix</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date de production</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date de récolte</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Origine</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabLotsPerimes">
          <attribute name="title">
           <string>Lots périmés</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidgetLotPerimes">
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>160</number>
             </attribute>
             <column>
              <property name="text">
               <string>Produit</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Producteur</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Quantité</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Unité</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Prix</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date de production</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date de récolte</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Origine</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pushButtonSupprLotPerimer">
               <property name="text">
                <string>Supprimer le(s) lot(s) périmé(s)</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogGestionLot</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogGestionLot</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
